gensim/utils.py

```diff
@@ -217,6 +217,10 @@ def any2unicode(text, encoding='utf8', errors='strict'):
     return unicode(text, encoding, errors=errors)
 to_unicode = any2unicode
 
+def call_on_class_only(*args, **kwargs):
+    """Warns when load methods are called on instance"""
+    logger.warn('This method should be called on a class object.')
```

Comment:
We don't want to be failing quietly (or, if logging not configured, completely in silence), ignoring the user's explicit request.

Exception is more appropriate.